tests/integration/test_api/test_config/test_logs_rotation/test_logs_size_option.py

```diff
@@ -0,0 +1,158 @@
+"""
+copyright: Copyright (C) 2015, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: This test checks that API properly initializes depending of the logs configuration.
+
+components:
+    - api
+
+suite: config
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-apid
+    - wazuh-modulesd
+    - wazuh-analysisd
+    - wazuh-execd
+    - wazuh-db
+    - wazuh-remoted
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/api/getting-started.html
+    - https://documentation.wazuh.com/current/user-manual/api/configuration.html
+
+tags:
+    - api
+"""
+
+
+import pytest
+from pathlib import Path
+from subprocess import CalledProcessError
+
+from . import CONFIGURATIONS_FOLDER_PATH, TEST_CASES_FOLDER_PATH
+from wazuh_testing.constants.api import CONFIGURATION_TYPES, WAZUH_API_PORT
+from wazuh_testing.utils.configuration import get_test_cases_data, load_configuration_template
+from wazuh_testing.constants.paths.logs import WAZUH_API_LOG_FILE_PATH
+from wazuh_testing.tools.monitors import file_monitor
+from wazuh_testing.utils import services
+from wazuh_testing.utils.callbacks import generate_callback
+from wazuh_testing.modules.api.patterns import API_STARTED_MSG
+
+# Marks
+pytestmark = pytest.mark.server
+
+# Variables
+# Used by add_configuration to select the target configuration file
+configuration_type = CONFIGURATION_TYPES[0]
+
+# Paths
+test_configuration_path = Path(CONFIGURATIONS_FOLDER_PATH, 'configuration_logs_rotation_config.yaml')
+test_cases_path = Path(TEST_CASES_FOLDER_PATH, 'cases_sizes_options.yaml')
+
+# Configurations
+test_configuration, test_metadata, test_cases_id = get_test_cases_data(test_cases_path)
+test_configuration = load_configuration_template(test_configuration_path, test_configuration, test_metadata)
+
+
+@pytest.mark.tier(level=0)
+@pytest.mark.parametrize('test_configuration,test_metadata', zip(test_configuration, test_metadata), ids=test_cases_id)
+def test_logs_rotation_size_option_values(test_configuration, test_metadata, add_configuration,
+                                          truncate_monitored_files, capfd):
+    """
+    description: Check if the API works as expected with different values of log rotation size
```

Comment:
```suggestion
    description: Check if the API works as expected with different values of log rotation size.
```

Reply:
Fixed typo in commit [bce3422].(https://github.com/wazuh/wazuh/pull/21776/commits/bce3422023e219c971baa91896a70c2a13678d31)